run_max_ent.py

```diff
@@ -0,0 +1,103 @@
+"""Script running the MaxEnt IRL
+algorithm on a trees data set.
+Usage:
+    python scripts/run_max_ent.py data=$PWD/path_to_trees.csv n_action=5
+Note:
+    The data path must be absolute not relative.
+"""
+import dataclasses
+from locale import normalize
+import logging
+from typing import Any, Dict, Optional
+
+import anytree
+from matplotlib.pyplot import get
+import pandas as pd
+import numpy as np
+
+import phirl.api as ph
+import phirl.hydra_utils as hy
+from phirl.maxent import *
```

Comment:
The syntax `from module import *` is considered a bad practice in modern Python (namespace collisions and harder dependencies tracking essentially: https://stackoverflow.com/a/2360808).